src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/client/transformation/TransformationClient.java

```diff
@@ -64,13 +64,18 @@ public SuccessfulTransformationResponse transformExceptionRecord(
             );
 
             if (ex.getStatusCode().equals(UNPROCESSABLE_ENTITY) || ex.getStatusCode().equals(BAD_REQUEST)) {
-                throw new InvalidCaseDataException(
-                    ex,
-                    objectMapper.readValue(ex.getResponseBodyAsByteArray(), TransformationErrorResponse.class)
-                );
+                try {
+                    TransformationErrorResponse error = objectMapper.readValue(
+                        ex.getResponseBodyAsByteArray(),
+                        TransformationErrorResponse.class
+                    );
+                    throw new InvalidCaseDataException(ex, error);
+                } catch (IOException e) {
+                    throw new InvalidCaseDataException(ex, null);
```

Comment:
Should not it be better to have a constructor of InvalidCaseDataException with 'String msg' parameter and use ex.getResponseBodyAsString() as this parameter? Rather than using null